spec/javascripts/Modules.AllocationDataTable_spec.js

```diff
@@ -105,7 +105,8 @@ describe('Modules.AllocationDataTable.js', function () {
         lengthMenu: 'Claims per page: _MENU_',
         emptyTable: '',
         infoFiltered: '',
-        processing: ''
+        processing: '',
+        paginate: Object({ previous: 'Previous', next: 'Next' })
```

Comment:
Is there a reason why this needs to be `Object({ previous: 'Previous', next: 'Next' })` whilst in `app/webpack/javascripts/modules/Modules.AllocationDataTable.js` is is just `{ previous: 'Previous', next: 'Next' }`?

Reply:
I just copied that from the failure message I got when I initially ran the tests 🙈 

```
1) Modules.AllocationDataTable.js ...Options ...should have `language`
  Message:
    Expected object not to have properties
        paginate: Object({ previous: 'Previous', next: 'Next' })
```

It works fine without the `Object(...)` wrapper too so I'll update the commit.